pages/blog-posts.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <section class="bg-mau-primary-50 h-full my-2">
+    <div class="container mx-auto">
+      <p
+        class="
+          font-medium
+          xl:text-4xl
+          text-2xl text-mau-primary-700
+          my-5
+          sm:mx-12
+          ml-10
+          md:text-left
+          text-center
+        "
+      >
+        Gotta Read This!
+      </p>
+      <div
+        class="
+          grid
+          md:grid-cols-2
+          2xl:grid-cols-3
+          px-24
+          sm:-mx-24
+          -mx-8
+          gap-4 gap-1
+        "
+      >
+        <div class="py-5" v-for="post of posts" :key="post.slug">
+          <NuxtLink :to="post.slug">
```

Comment:
This Link, in the preview is bigger than the actual card. (see comment in CardsAdvancedthumbnail on why)